.github/workflows/build_and_test.yml

```diff
@@ -3822,6 +3855,8 @@ jobs:
     - name: extract OpenDDS artifact
       shell: bash
       run: |
+        SSL_DIR=$PWD/openssl_install
+        export SSL_LIBDIR=$SSL_DIR/lib64
```

Comment:
I don't think this does anything.

```suggestion
```

Reply:
SSL_LIBDIR needs to be defined and exported when `configure` runs because it must be defined for mpc.

Reply:
Didn't realize I duplicated it.